# Parameters for placing the odom frame in the world frame
# Be sure if you type an integer and want a floating point, use #.0
odom_transform:
    x: 0.0
    y: 0.0
    z: 0.0
    qx: 0.0
    qy: 0.0
    qz: 0.0
    qw: 1.0
    frame_id: "world"
    child_frame_id: "odom"
    period_in_ms: 100.0
